### PR TITLE
[15.0][MIG] l10n_th_withholding_tax

### PR DESCRIPTION
**Pull request by kurogeek**

### CODE REVIEW

**Comment by kurogeek** [comment]
Please tell me if you guys want me to fix the pylint errors.

**Comment by kittiu** [comment]
@kurogeek thanks for help doing migration.
I found that v14 modules (mainly accounting related modules) are too modular and too complex.
It was ok to use, but I found it less complex and support more use cases when combined. So, in our project, we are using this now --> https://github.com/OCA/l10n-thailand/pull/272

And I am planning to [ADD] them into v15, Some work already started, I will to re-open and continue it. Then, you can help me test it? :)

* https://github.com/OCA/l10n-thailand/pull/279
* https://github.com/OCA/l10n-thailand/pull/280
* https://github.com/OCA/l10n-thailand/pull/281
* https://github.com/OCA/l10n-thailand/pull/283
* https://github.com/OCA/l10n-thailand/pull/284

In summary, v15, for Thai accounting will be fresh start with l10n_th_account_xxx

And so, l10n_th_withholding_tax won't be in v15, as it is not needed and will conflict with the rest too.

**Comment by kittiu** [comment]
@kurogeek not sure you get what I try to say. We can have an online meeting if you want.

**Comment by kurogeek** [comment]
Okay, I got what you are saying and understood. I can help you test them if it makes the process faster.
